composer.json

```diff
@@ -20,7 +20,6 @@
         "symfony/framework-bundle": "~2.5|~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8",
```

Comment:
PHPUnit isn't compatible with Symfony 3.1: https://github.com/phpspec/prophecy/pull/275

So we have two options:
- remove Symfony 3.1 from Travis
- remove PHPUnit from dev deps

Since PHPUnit is usually always installed on a dev machine (and also on Travis) I've chosen the second one.

What is your choice? :)